Code/Tools/ProjectManager/Source/Settings.cpp

```diff
@@ -167,15 +168,16 @@ namespace O3DE::ProjectManager
         bool success = GetBuiltSuccessfullyPaths(builtPathsResult);
 
         AZStd::string projectPath = projectInfo.m_path.toStdString().c_str();
+        AZ::IO::FixedMaxPath lexicallyNormalPath = AZ::IO::FixedMaxPath(projectPath).LexicallyNormal();
         if (successfullyBuilt)
         {
             //Add successfully built path to set
-            builtPathsResult.insert(projectPath);
+            builtPathsResult.insert(lexicallyNormalPath.c_str());
         }
         else
         {
             // Remove unsuccessfully built path from set
-            builtPathsResult.erase(projectPath);
+            builtPathsResult.erase(lexicallyNormalPath.c_str());
```

Comment:
Changing the lexicallyNormalPath type to an AZ::IO::Path will avoid the need of the `.c_str()` which prevents an additional construction of an AZ::IO::Path(and underlying AZStd::string member on it).

Reply:
I'm going to leave the Settings changes out of this PR because they are not related to the engine changes and a little extra work is necessary to make sure the settings are backwards compatible.  Tracking with https://github.com/o3de/o3de/issues/10948